activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -138,6 +138,15 @@ def indexes(table_name) # :nodoc:
               end
             end
 
+            with_params = {}
+            if with
+              with.split(", ").each do |w|
+                k, v = w.split("=", 2)
+                v = v[1..-2] if v.start_with?("'")
+                with_params[k] = v
```

Comment:
This presumably needs to properly unquote. (Do we already know how to do that for default values or something?)